runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -330,6 +326,8 @@
     @Getter
     private NodeRouterPool nodeRouterPool;
 
+    public volatile long trimMark = Address.NON_ADDRESS;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make trimMark a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)